scripts/filter_language_issues.py

```
@@ -0,0 +1,181 @@
+        return []
+
+
+def main():
+    """Run filtered LanguageTool check on main documentation."""
+    print("Running filtered LanguageTool check...")
+
+    # Initialize LanguageTool
+    try:
+        tool = language_tool_python.LanguageTool("en-US")
+        print("Using LanguageTool local server")
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)